README.md

```diff
@@ -29,7 +29,7 @@ Firstly, clone this GitHub repository. We'll apply files to `kubectl` from this
 We apply these files as such:
 
```

Comment:
We also need to add some description that we want users to install CNI plugin for k8s cluster network because previous version we expect that user don't install any CNI plugin, as well as you mentioned that `doc/quickstart.md`

Reply:
Thanks for the eye on this one, totally missed that paragraph in the main README, appreciate it.